build/org.eclipse.cdt.managedbuilder.core/src/org/eclipse/cdt/managedbuilder/internal/core/jsoncdb/generator/CompilationDatabaseGenerator.java

```diff
@@ -414,4 +426,41 @@ public boolean visit(IResourceProxy proxy) throws CoreException {
 
 	}
 
+	private String findCompilerInPath(ITool tool, IConfiguration config) {
+		if (toolMap.containsKey(tool)) {
+			return "\"" + toolMap.get(tool) + "\""; //$NON-NLS-1$//$NON-NLS-2$
+		}
+		String compilerName = CompilationDatabaseGenerator.getCompilerName(tool);
+		File pathToCompiler = new File(compilerName);
+
+		if (pathToCompiler.isAbsolute()) {
+			toolMap.put(tool, compilerName);
+			return "\"" + compilerName + "\""; //$NON-NLS-1$ //$NON-NLS-2$
+		}
+		ICConfigurationDescription cfg = ManagedBuildManager.getDescriptionForConfiguration(config);
+		IEnvironmentVariable[] variables = CCorePlugin.getDefault().getBuildEnvironmentManager().getVariables(cfg,
+				true);
+
+		for (IEnvironmentVariable variable : variables) {
+			if ("PATH".equals(variable.getName())) { //$NON-NLS-1$
```

Comment:
Is PATH guaranteed to be uppercase in return from getVariables? Normally on Windows we have to handle the case of path being case insensitive.

Reply:
Fixed in latest version with equalsIgnoreCase